app/webpacker/components/EditSchedule/EditActivities/index.js

```diff
@@ -101,15 +107,14 @@ function EditActivities({
       const matchCount = getMatchingActivities(wcifSchedule, activity).length - 1;
       const matchesText = ` (${matchCount} matching activit${matchCount === 1 ? 'y' : 'ies'})`;
 
+      const fcTitle = activityToFcTitle(activity) + (shouldUpdateMatches && matchCount > 0 ? matchesText : '');
+
       return {
-        title: activity.name + (shouldUpdateMatches && matchCount > 0 ? matchesText : ''),
+        title: fcTitle,
         start: activity.startTime,
         end: activity.endTime,
         extendedProps: {
-          activityId: activity.id,
-          activityCode: activity.activityCode,
-          activityName: activity.name,
-          childActivities: activity.childActivities,
+          activity,
```

Comment:
If `activityPickerRef` is only providing a partial activity, would the key make more sense as `partialActivity`?
```suggestion
          partialActivity: activity,
```

Reply:
I don't think so, because then our helper coe who is able to turn _any_ fcEvent into an activity won't be able to know which key it has to access (might be `activity` might be `partialActivity`)

Reply:
Hm. My concern stems from `activity: partialActivity` on line 142 below, which feels wrong. I was thinking every `activity` key would have to be switched to `partialActivity`, since it seems like sometimes the activity is partial and sometimes is full. (I would say a full activity is a partial activity - but not vice-versa of course.)

Reply:
Attended to the review comments by introducing a shared key across the whole FC integration :smile: